kubernetes-client/src/main/java/io/fabric8/kubernetes/client/informers/SharedScheduler.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.fabric8.kubernetes.client.informers;
+
+import io.fabric8.kubernetes.client.utils.Utils;
+
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Maintains a single thread daemon scheduler.
+ * It is not intended for long-running tasks
+ */
+public class SharedScheduler {
+  
+  public static long DEFAULT_TTL_MILLIS = TimeUnit.SECONDS.toMillis(10);
```

Comment:
Do you think 10 milliseconds is too short for TTL or is it fine?

Reply:
That actually evaluates to 10 seconds worth of millis.